tst/EnergyPlus/unit/SimAirServingZones.unit.cc

```diff
@@ -0,0 +1,170 @@
+// EnergyPlus::SimAirServingZones Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// C++ Headers
+#include <cassert>
+#include <cmath>
+#include <string>
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array.functions.hh>
+#include <ObjexxFCL/Fmath.hh>
+#include <ObjexxFCL/gio.hh>
+
+// EnergyPlus Headers
+#include <SimAirServingZones.hh>
+#include <BranchInputManager.hh>
+#include <DataAirLoop.hh>
+#include <DataAirSystems.hh>
+#include <DataContaminantBalance.hh>
+#include <DataConvergParams.hh>
+#include <DataDefineEquip.hh>
+#include <DataEnvironment.hh>
+#include <DataGlobals.hh>
+#include <DataHeatBalance.hh>
+#include <DataHVACControllers.hh>
+#include <DataHVACGlobals.hh>
+#include <DataLoopNode.hh>
+#include <DataPrecisionGlobals.hh>
+#include <DataSizing.hh>
+#include <DataSystemVariables.hh>
+#include <DataZoneEquipment.hh>
+#include <DesiccantDehumidifiers.hh>
+#include <EMSManager.hh>
```

Comment:
@rongpengzhang I understand it is a slight time savings, but really you should strive to only include what you need to include. 

Personally, I think everyone should be explicit about what they need and don't need for unit testing. The unit tests should test very specific functionality and throughly cover all possible valid AND invalid code paths. This is how we build up a large test suite and once that happens we can have much greater confidence when refactoring code to avoid regressions.

Commenting out the includes you do not need for your unit tests works, but I would still push my opinion to remove them entirely.